lib/ext/ws/event.ex

```diff
@@ -0,0 +1,8 @@
+defmodule Ext.Ws.Event do
+  @derive {Jason.Encoder, except: [:__meta__, :__struct__]}
+
+  defstruct [:uuid, :data]
+
+  def init(%{uuid: uuid, data: data}), do: %__MODULE__{uuid: uuid, data: data}
```

Comment:
looks like this is enough 

```
def init(%__MODULE__{} = struct), do: struct
```

Reply:
from redis we get map, not struct

Reply:
why we can not add to `first` sctruct?

Reply:
actually, I see two cases:
1. We use first
2. we pass data as is and should add uuid to this map